src/main/java/org/jenkinsci/plugins/electricflow/extension/impl/CloudBeesFlowPipelineRest.java

```diff
@@ -9,7 +9,7 @@
 import org.jenkinsci.plugins.variant.OptionalExtension;
 import org.jenkinsci.plugins.workflow.job.WorkflowRun;
 
-@OptionalExtension(requirePlugins = "pipeline-rest-api")
+@OptionalExtension(requirePlugins = {"pipeline-rest-api","workflow-job"})
```

Comment:
https://github.com/jenkinsci/electricflow-plugin/pull/311/files#diff-5fe4dcab045ed4e397c90b2207b14cf8d791b70c7a5f5db9cab0255acf95244eR19 used `WorkflowRun` yet the dependency was not added here relying on the transitive dependency of `pipeline-rest-api`.